src/typescript/frontend/src/app/market/[market]/page.tsx

```diff
@@ -83,7 +83,7 @@ const EmojicoinPage = async (params: EmojicoinPageProps) => {
       fetchSwapEvents({ marketID, pageSize: EVENTS_ON_PAGE_LOAD }),
       wrappedCachedContractMarketView(addresses.marketAddress.toString()),
       getAptPrice(),
-      fetchAllFungibleAssetsBalance({ max: 100, assetType: addresses.emojicoin.toString() }),
+      fetchHolders(addresses.emojicoin.toString()),
```

Comment:
```suggestion
      fetchHolders(marketAddress),
```

and above, (github won't let me edit it)

change this:

```typescript
const addresses = getEmojicoinMarketAddressAndTypeTags({ symbolBytes: encodeEmojis(emojis) });
// to this:
const marketAddress = getMarketAddress(emojis).toString();
```